static/app/views/automations/edit.tsx

```diff
@@ -116,6 +116,15 @@ function AutomationEditForm({automation}: {automation: Automation}) {
   const [automationBuilderErrors, setAutomationBuilderErrors] = useState<
     Record<string, string>
   >({});
+  const [updateErrors, setUpdateErrors] =
+    useState<RequestError['responseJSON']>(undefined);
+
+  const {mutateAsync: updateAutomation} = useUpdateAutomation({
+    onError: error => {
+      setUpdateErrors(error.responseJSON);
+    },
+  });
```

Comment:
we don't need an additional state here, we should be able to use the error object on the mutation
```suggestion
  const {mutateAsync: updateAutomation, error} = useUpdateAutomation();
```